forem/forem
# Downgrade binding_of_caller to fix Docker compose

## What type of PR is this? (check all applicable)


## Description

Docker compose is broken because of `binding_of_caller`, this fixes it

Closes https://github.com/forem/forem/issues/12068

## Related Tickets & Documents

- https://github.com/forem/forem/issues/12068
- #12059 

## QA Instructions, Screenshots, Recordings

Run `bin/container-setup` and check it doesn't break with `NameError: uninitialized constant RubyVM::DebugInspector`

## Added tests?

- [x] No, and this is why: gem change to fix Docker setup

## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
